src/main/java/liquibase/nosql/snapshot/NoSqlSnapshotGenerator.java

```diff
@@ -0,0 +1,41 @@
+package liquibase.nosql.snapshot;
+
+import liquibase.database.Database;
+import liquibase.exception.DatabaseException;
+import liquibase.ext.opensearch.database.OpenSearchLiquibaseDatabase;
+import liquibase.snapshot.DatabaseSnapshot;
+import liquibase.snapshot.InvalidExampleException;
+import liquibase.snapshot.SnapshotGenerator;
+import liquibase.snapshot.SnapshotGeneratorChain;
+import liquibase.structure.DatabaseObject;
+
+import java.util.ResourceBundle;
+
+import static liquibase.plugin.Plugin.PRIORITY_SPECIALIZED;
+
+public class NoSqlSnapshotGenerator implements SnapshotGenerator {
+    private static final ResourceBundle resourceBundle = ResourceBundle.getBundle("liquibase/i18n/liquibase-opensearch"); // TODO: not generic
```

Comment:
@filipelautert: should i just remove these TODOs for the time being? i anyway didn't add them in all places